tabbycat/api/urls.py

```diff
@@ -7,7 +7,19 @@
 
 urlpatterns = [
 
+    path('',
+        views.APIRootView.as_view(),
+        name='api-root'),
+
+    path('create/',
+        views.TournamentCreateView.as_view(),
+        name='api-tournament-create'),
+
     path('<slug:tournament_slug>/', include([
+        path('',
+            views.TournamentDetailView.as_view(),
+            name='api-tournament-detail'),
+
         path('<int:round_seq>/', include([
```

Comment:
Maybe prefix with `round/`?

Reply:
Maybe! The answer will probably be clearer once we start actually implementing round-specific APIs.

Reply:
And with the current discussion with the `/tournament/` prefix.